docs/en/development/continuous-integration.md

```diff
@@ -71,7 +71,7 @@ If it fails, fix the style errors following the [code style guide](style.md).
 ```sh
 mkdir -p /tmp/test_output
 # running all checks
-docker run --rm --volume=.:/ClickHouse --volume=/tmp/test_output:/test_output -u $(id -u ${USER}):$(id -g ${USER}) --cap-add=SYS_PTRACE clickhouse/style-test
+python3 tests/ci/style_check.py --no-push
 
 # run specified check script (e.g.: ./check-mypy)
```

Comment:
could be removed till the end of Style check I think

Reply:
Sorry, I don't get it. It's the doc for https://clickhouse.com/docs/en/development/continuous-integration#running-style-check-locally

What could be removed?

Reply:
from `# run specified check script (e.g.: ./check-mypy)` and till the end of style check. too detailed.
never mind, maybe next time...

Reply:
Unless we add arguments to run them from `style_check.py,` it's useful info for external devs. After that yes, the docker workarounds can go